react-native-libraries.json

```
@@ -9762,5 +9762,25 @@
     "web": true,
     "expo": true,
     "template": true
+  },
+  {
+    "githubUrl": "https://github.com/troberts-28/react-native-timer-picker",
+    "npmPkg": "react-native-timer-picker",
+    "examples": [
+      "https://github.com/troberts-28/react-native-timer-picker#timer-picker-modal-dark-mode-",
+      "https://github.com/troberts-28/react-native-timer-picker#timer-picker-modal-light-mode-",
+      "https://github.com/troberts-28/react-native-timer-picker#timer-picker-with-customisation-dark-mode-",
+      "https://github.com/troberts-28/react-native-timer-picker#timer-picker-with-customisation-light-mode-"
+    ],
```

```suggestion
    "examples": [
      "https://github.com/troberts-28/react-native-timer-picker/tree/main/example"
    ],
```

Let's link to the example app instead of readme sections.